golang/cosmos/x/lien/spec/02_messages.md

```diff
@@ -39,15 +39,42 @@ Response:
 
 See [Concepts](01_concepts.md) for the meaning of the amounts.
 
-## From Golang to Javascript
+### LIEN_GET_STAKING
 
-### LIEN_SLASHED
+* type: "LIEN_GET_STAKING"
+* validators: array of strings of bech32-encoded operator addresses
+* delegators: array of strings of bech32-encoded account addresses
 
-* type: "LIEN_SLASHED"
-* addresses: array of bech32-encoded strings, lexicographically sorted
-* currentTime: string timestamp
+Response:
+
+* epoch_tag: string encoding the staking epoch.
+If it is the same in two different responses, the results can be
+safely aggregated.
+* denom: string giving the name of the staking token
+* validator_values: array of string encoding nonnegative integer
+giving the tokens delegated to the validator of the same index
+as the request, or `"-1"` for a malformed validator address.
+* delegator_states: array of objects corresponding to the delegators
+in the request, where the object contains:
```

Comment:
I would slightly prefer `null` values for the entire errored structure instead of `"-1"` sentinels within the structure, but I'm not sure how easy that is to implement.

Reply:
It's not too bad - and worth it to make the JSON representation more idiomatic. Good idea. Done.